compliance_checker/cf/cf.py

```diff
@@ -1455,7 +1457,7 @@ def check_add_offset_scale_factor_type(self, ds):
             results.extend(
                 list(
                     map(
-                        lambda x: self._check_scale_factor_add_offset(
+                        lambda x: self._check_add_offset_scale_factor(
```

Comment:
This one doesn't exist either. Do you mean `_check_add_offset_scale_factor_type`? (the function defined directly above this one?

Reply:
Erroneous function name has been addressed.